packages/tree-view/src/TreeViewItem.ts

```diff
@@ -94,9 +94,19 @@ export class TreeViewItem extends ObserveSlotPresence(
         this.open = !this.open;
     }
 
-    public toggleSelected(event: Event): void {
+    public toggleSelected(event: MouseEvent): void {
         event.preventDefault();
         this.selected = !this.selected;
+        const multiselect = event.shiftKey || event.metaKey;
+        const applyDefault = this.dispatchEvent(
```

Comment:
This currently dispatches `selected` and `deselected` events as a reaction to the change to `this.selected` as well: https://github.com/adobe/spectrum-web-components/pull/1423/files#diff-b84c849cef36d2ec108a54310d7b0c4b3989e7bc837147666dad0e5a4ec5e50fR157 should be one of the other, IMO.